packages/localizations/src/el-GR.ts

```diff
@@ -0,0 +1,692 @@
+      formTitle: 'Κωδικός επαλήθευσης',
+      formSubtitle: 'Εισαγάγετε τον κωδικό επαλήθευσης που απεστάλη στο τηλέφωνό σας',
+      resendButton: 'Δεν λάβατε κωδικό; Αποστολή ξανά',
+    },
+  },
+} as const;
+
+export const elGR: LocalizationResource = {
+  locale: 'el-GR',
+  socialButtonsBlockButton: 'Συνέχεια με {{provider|titleize}}',
+  dividerText: 'ή',
+  formFieldLabel__emailAddress: 'Διεύθυνση email',
+  formFieldLabel__emailAddresses: 'Διευθύνσεις email',
+  formFieldLabel__phoneNumber: 'Αριθμός τηλεφώνου',
+  formFieldLabel__username: 'Όνομα χρήστη',
+  formFieldLabel__emailAddress_phoneNumber: 'Διεύθυνση email ή αριθμός τηλεφώνου',
+  formFieldLabel__emailAddress_username: 'Διεύθυνση email ή όνομα χρήστη',
+  formFieldLabel__phoneNumber_username: 'Αριθμός τηλεφώνου ή όνομα χρήστη',
+  formFieldLabel__emailAddress_phoneNumber_username: 'Διεύθυνση email, αριθμός τηλεφώνου ή όνομα χρήστη',
+  formFieldLabel__password: 'Κωδικός πρόσβασης',
+  formFieldLabel__currentPassword: 'Τρέχων κωδικός πρόσβασης',
+  formFieldLabel__newPassword: 'Νέος κωδικός πρόσβασης',
+  formFieldLabel__confirmPassword: 'Επιβεβαίωση κωδικού πρόσβασης',
+  formFieldLabel__signOutOfOtherSessions: 'Αποσύνδεση από όλες τις άλλες συσκευές',
+  formFieldLabel__firstName: 'Όνομα',
+  formFieldLabel__lastName: 'Επώνυμο',
+  formFieldLabel__backupCode: 'Αντίγραφο ασφαλείας κωδικού',
+  formFieldLabel__organizationName: 'Όνομα οργανισμού',
+  formFieldLabel__organizationSlug: 'Συντόμευση URL',
+  formFieldLabel__confirmDeletion: 'Επιβεβαίωση',
+  formFieldLabel__role: 'Ρόλος',
+  formFieldInputPlaceholder__emailAddress: '',
+  formFieldInputPlaceholder__emailAddresses:
+    'Εισαγάγετε ή επικολλήστε μία ή περισσότερες διευθύνσεις email, χωρισμένες με κενά ή κόμματα',
+  formFieldInputPlaceholder__phoneNumber: '',
+  formFieldInputPlaceholder__username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber: '',
+  formFieldInputPlaceholder__emailAddress_username: '',
+  formFieldInputPlaceholder__phoneNumber_username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber_username: '',
+  formFieldInputPlaceholder__password: '',
+  formFieldInputPlaceholder__firstName: '',
+  formFieldInputPlaceholder__lastName: '',
+  formFieldInputPlaceholder__backupCode: '',
+  formFieldInputPlaceholder__organizationName: '',
+  formFieldInputPlaceholder__organizationSlug: '',
+  formFieldError__notMatchingPasswords: 'Οι κωδικοί δεν ταιριάζουν.',
+  formFieldError__matchingPasswords: 'Οι κωδικοί ταιριάζουν.',
+  formFieldAction__forgotPassword: 'Ξεχάσατε τον κωδικό;',
+  formFieldHintText__optional: 'Προαιρετικό',
+  formButtonPrimary: 'Συνέχεια',
+  signInEnterPasswordTitle: 'Εισαγωγή κωδικού πρόσβασης',
+  backButton: 'Πίσω',
+  footerActionLink__useAnotherMethod: 'Χρησιμοποιήστε άλλη μέθοδο',
+  badge__primary: 'Κύριο',
+  badge__thisDevice: 'Αυτή η συσκευή',
+  badge__userDevice: 'Συσκευή χρήστη',
+  badge__otherImpersonatorDevice: 'Άλλη συσκευή αντιπροσώπου',
```

Comment:
@EmmanouelaPothitou Not sure if "αντιπροσώπου" is right here. This translates to "representative". 
I think "Άλλη συσκευή υποδυόμενου" is better. WDYT?